docs/admin/daemons.md

```diff
@@ -209,9 +209,12 @@ all or certain hosts, and when it needs to start before other pods.
 
 ## Caveats
 
-DaemonSet is part of the experimental API group, so it is not subject to the same compatibility
-guarantees as objects in the main API.  It may not be enabled.  Enable by setting
-`--runtime-config=experimental/v1alpha1` on the apiserver.
+DaemonSet objects are in the [`experimental` API Group](../api.md#api-groups).  This API group may or
+may not be enabled on your cluster.
+
+DaemonSet objects have [API version `v1alpha1`](../api.md#api-versioning).  Alpha objects may change
```

Comment:
v1beta1, but they're turned off by default until they actually meet the beta bar.